src/utils/tests/DevToolsUtils.spec.js

```diff
@@ -0,0 +1,35 @@
+import { reportException, executeSoon } from "../DevToolsUtils.js";
+
+describe("DevToolsUtils", () => {
+  describe("reportException", () => {
+    beforeEach(() => {
+      global.console = { error: jest.fn() };
+    });
```

Comment:
do we need to unmock these in an `afterEach`?

Reply:
closing because this works.

Reply:
Thanks for the tip, Ill keep that in mind for the future